### ISSUE TITLE
"This schematic format is not recognised or supported" since latest update

### ISSUE THREAD

**Issue by Geitenijs** [IntellectualSites/PlotSquared#4478]
### Server Implementation

Paper

### Server Version

1.21.1

### Describe the bug

Without any modifications to my setup or schematics, after updating from 1.20.6 and P2 7.3.8 to 1.21.1 and P2 7.3.9 I'm getting the following error during startup. I'm running the latest version of FAWE (#879) if that matters.

### To Reproduce

See description.

### Expected behaviour

No error, schematic was working just fine before the update.

### Screenshots / Videos

[Schematics.zip](https://github.com/user-attachments/files/16581009/Schematics.zip)

### Error log (if applicable)

https://paste.gg/p/anonymous/45fb6cf8663e48289ce5c1a2a46a51f8

### Plot Debugpaste

https://athion.net/ISPaster/paste/view/c12821b1857841e899885a398f35d21a

### PlotSquared Version

7.3.9-Premium

### Checklist

- [X] I have included a Plot debugpaste.
- [X] I am using the newest build from https://www.spigotmc.org/resources/77506/ and the issue still persists.

**Comment by PierreSchwang** [comment]
I see, makes sense. FAWE updated the default schematic format to Sponge v3 whereas your schematic(s) were most likely generated with v2. The current implementation has the format statically set (and now statically set to v3). That should be dynamically be determined by isFormat checks.

**Comment by Bobcat00** [comment]
I'm seeing the same thing. I don't use schematics, so I assume it's PlotSquared's own schematic for world/road generation.

**Comment by PierreSchwang** [comment]
@Geitenijs Might it be possible, that you've updated FAWE in the same process of updating PS and your server?

**Comment by Geitenijs** [comment]
@PierreSchwang Yeah I did! I updated all relevant plugins to their latest version today.

**Comment by PierreSchwang** [comment]
Will most likely correlate to https://github.com/IntellectualSites/PlotSquared/issues/4460

The underlying change is the schematic rework - PS statically writes sponge v1 / v2 schematics but often with the wrong extension which might be the root cause. Needs quite a few larger updates to get it running again with supporting the older formats.

**Comment by Bryson0518** [comment]
This problem is happening on my server as well however I am using a template for the pathway
